examples/arm/zlib.BUILD.bazel

```
@@ -0,0 +1,25 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+config_setting(
+    name = "is_linux_aarch64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:aarch64",
+    ],
+)
+
+filegroup(
+    name = "include",
+    srcs = glob(["include/*.h"]),
+    visibility = ["//visibility:public"],
+)
+cc_library(
+    name = "zlib",
+	srcs = select({
+	    "//:is_linux_aarch64": ["@nixpkgs_zlib_arm//:lib"],
+        "//conditions:default": ["@nixpkgs_zlib_x86//:lib"],
```

```suggestion
        "//:is_linux_aarch64": ["@nixpkgs_zlib_arm//:lib"],
        "//conditions:default": ["@nixpkgs_zlib_x86//:lib"],
```
Running `buildifer -r` over this directory is an easy way to fix all formatting issues.

---

Buildifier is not fixing this one. I'm running both
```
nix-shell --pure --run "bazel run --disk_cache=~/bazel-disk-cache //:buildifier -- -r examples/arm"
nix-shell --pure --run "bazel run --disk_cache=~/bazel-disk-cache //:buildifier-fix"
```
Both commands succeed without changing or reporting any files.

---

They do catch formatting issues on the WORKSPACE file (if I introduce them). But they seem to ignore `examples/arm/zlib.BUILD.bazel `

---

Oh, I see. Sorry, in this case, it needs to be called directly on the file, given that it's not a standard BUILD file name.

---

I renamed the file to `BUILD.zlib.bazel`, buildifier finds it now.